ui/app/templates/components/form-field.hbs

```diff
@@ -88,6 +88,12 @@
     inputValue=(get model valuePath)
     onChange=(action (action "setAndBroadcast" valuePath))
   }}
+{{else if (eq attr.options.sensitive true)}}
+  {{masked-input
+    isPassword=true
+    value=(or (get model valuePath) attr.options.defaultValue)
+    displayOnly=false
```

Comment:
I think `false` is the default for `displayOnly` so you don't need to pass it here.